tests/run-tests.sh

```diff
@@ -443,6 +443,14 @@ check_exit_code() {
     fi
 }
 
+# verify that yum is updated
+case ${distro} in
+    Ubuntu*) sudo apt-get -q --assume-no update;;
+    CentOS* | AlmaLinux*) sudo yum update -n;;
```

Comment:
pls use --setopt tsflags=test flag
`yum update -y --setopt tsflags=test`
and then clean the cache using `yum clean packages`